python/nano/src/bigdl/nano/pytorch/utils.py

```diff
@@ -144,24 +145,26 @@ def load_model(path, model: pl.LightningModule = None, input_sample=None,
     with open(meta_path, 'r') as f:
         metadata = yaml.safe_load(f)
     model_type = metadata.get('ModelType', None)
+    result = None
     if model_type == 'PytorchOpenVINOModel':
-        invalidInputError(model is None,
-                          "Argument 'model' must be None for OpenVINO loading.")
-        return load_openvino_model(path, device=device)
+        result = load_openvino_model(path, device=device)
     if model_type == 'PytorchONNXRuntimeModel':
-        invalidInputError(model is None,
-                          "Argument 'model' must be None for ONNX Runtime loading.")
-        return load_onnxruntime_model(path)
+        result = load_onnxruntime_model(path)
     if model_type == 'PytorchQuantizedModel':
-        return load_inc_model(path, model, 'pytorch', input_sample=input_sample)
+        result = load_inc_model(path, model, 'pytorch', input_sample=input_sample)
     if model_type == 'PytorchIPEXJITModel':
-        return load_ipexjit_model(path, model, inplace=inplace)
+        result = load_ipexjit_model(path, model, inplace=inplace)
     if model_type == 'PytorchIPEXJITBF16Model':
-        return load_ipexjitbf16_model(path, model, inplace=inplace)
+        result = load_ipexjitbf16_model(path, model, inplace=inplace)
     if model_type == 'PytorchIPEXQuantizationModel':
-        return load_ipex_quantization_model(path, model, inplace=inplace)
+        result = load_ipex_quantization_model(path, model, inplace=inplace)
     if model_type == 'BF16Model':
         return load_bf16_model(path, model)
+    if result is not None:
```

Comment:
may give some error message if `result is None`

Reply:
I think should not raise error here, as if result is None, it maybe a normal nn.Module and it will enter below logic.

Reply:
yes, you're right